src/Otis.Users/UserRegisterRequestModel.cs

```diff
@@ -0,0 +1,18 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Otis.Users;
+
+public sealed class UserRegisterRequestModel
```

Comment:
I would say that the only required field is email. Eventually first name also, because I have been into context where people write everything in the same field or they are just lazzy and put the first name and that is is. If they do not input their last name is not problematic, I would say we should make mandatory only first name and email to reduce the probablity of capturing pre-registrations

Reply:
Agreed. I'll update the code and the lambda to make FirstName and EmailAddress required and LastName optional

Reply:
`Lastname` is now optional. You can either pass it as `null` or not pass it at all and it works.
Lambda has been updated with this version